esmvaltool/diag_scripts/crem/ww09_esmvaltool.py

```diff
@@ -218,25 +218,22 @@ def main(cfg):
 
     # plot results
 
-    if cfg['write_plots']:
-        plotname = os.path.join(
-            cfg['plot_dir'],
-            'ww09_metric_multimodel.' + cfg['output_file_type'],
-        )
-        logger.debug("Plotting results to %s", plotname)
+    plotname = os.path.join(
```

Comment:
It might be nice to use `esmvaltool.diag_scripts.shared.get_plot_filename` instead